docs/forms/FORM_ERRORS.md

```diff
@@ -100,38 +78,45 @@ export interface ErrorMessageObject {
    * @example
    * ```typescript
    * {required: true}
-   * ```
+   * \```
    * @example
    * ```typescript
    * {max: {max 12, actual: 31}}
    * ```
    */
   validationError?: {[key: string]: any};
 }
-```
+`````
 
-<a name="createerror"></a>
+You can also find these interfaces [here](https://github.com/AmadeusITGroup/otter/blob/main/packages/%40o3r/forms/src/core/errors.ts).
 
-### Creating error object
+### Creating an error object
 
-The presenter has to implement the [Validator](https://angular.io/api/forms/NG_VALIDATORS) or [AsyncValidator](https://angular.io/api/forms/NG_ASYNC_VALIDATORS) in order to give us the possibility to define the error object which will be returned by the form.
-The error message structure will be defined in the implementation of __validate__ method.
-As __validate__ function should return a [ValidationErrors](https://angular.io/api/forms/ValidationErrors) object, which is a map of custom objects (with type _any_), we can prepare the returned object for the store of error messages. This will ease the process of adding the errors in the store.
-We have to make sure that we are providing the __htmlElementId__ for the errors in the store which is matching the __html field__.
-For this, the presenter is receiving an __id__ as input and for each field we are concatenating the __id__ with the __formControlName__. As the container is setting a __unique id__ we are sure that we have uniques html ids for the form fields.
-The object returned by the __validate__ is the error object which is propagated to the container.
+The presenter has to implement the [Validator](https://angular.io/api/forms/NG_VALIDATORS) or the [AsyncValidator](https://angular.io/api/forms/NG_ASYNC_VALIDATORS) interface
+in order to give us the possibility of defining the error object that will be returned by the form.
 
-There are 2 types of validators (see [Form Validation](./FORM_VALIDATION.md)), 2 categories of error messages:
+The error message structure will be defined in the implementation of the `validate` function.
 
-- one for __custom errors__ - set on the container
-- one for __primitive errors__ - computed in the presenter.
+As the `validate` function should return a [ValidationErrors](https://angular.io/api/forms/ValidationErrors) object, which is a map of custom objects (of type `any`),
+we can prepare the returned object for the store of error messages. This will ease the process of adding the errors in the store.
```

Comment:
Not sure about this one:
```suggestion
we can adapt the returned object to the store of error messages. This will ease the process of adding the errors in the store.
```